ccd-definition/AuthorisationCaseEvent/systemUpdate-nonprod.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
Is this going under the prod-nonprod release plan. Or directly merged to master?